internal/cloudapi/v2/handler.go

```diff
@@ -689,6 +689,10 @@ func imageStatusFromKojiJobStatus(js *worker.JobStatus, initResult *worker.KojiI
 		return ImageStatusValueFailure
 	}
 
+	if buildResult.OSBuildOutput != nil && !buildResult.OSBuildOutput.Success {
```

Comment:
Isn't this covered by `if buildResult.JobError`?

Reply:
Better be safe than sorry? I guess we can remove it when we will get to cleaning up job error handling (this will surely happen at some point).